waspls/README.md

```diff
@@ -0,0 +1,32 @@
+# Waspls
+
+This directory contains source code of the `wasp` language server (aka `waspls`)
+and this README is aimed at the contributors to the project.
+
+## Overview
+
+`waspls` is implemented in Haskell. It depends on `waspc` for parsing and
```

Comment:
Ok sounds good -> maybe just drop a TODO in waspls.cabal that shortly says it might make sense to put this into waspc cabal project and that is it for now.